
# HG changeset patch
# User Stuart Rackham <srackham@methods.co.nz>
# Date 1243888214 -43200
# Node ID cbfb5fb8d05ff88efd45477766c51aa95059dcd5
# Parent 0247d70c18d2f53d387cbb3c83ab3e356c2ed49a
Missing language file generates a warning instead of an error.

--- a/asciidoc.py	Thu May 28 13:28:36 2009 +1200
+++ b/asciidoc.py	Tue Jun 02 08:30:14 2009 +1200
@@ -1201,7 +1201,7 @@
         message.linenos = linenos
         if lang:
             if not config.load_lang(lang):
-                message.error('missing language conf file: lang-%s.conf' % lang)
+                message.warning('missing language conf file: lang-%s.conf' % lang)
             self.attributes['lang'] = lang  # Reinstate new lang attribute.
         else:
             message.error('language attribute (lang) is not defined')

